apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-patterns-plugin.tsx

```diff
@@ -4,19 +4,19 @@
 import '@wordpress/nux';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { addFilter, removeFilter } from '@wordpress/hooks';
-import { LayoutDefinition, PageTemplateModal } from '@automattic/page-template-modal';
+import { PagePatternModal, PatternDefinition } from '@automattic/page-pattern-modal';
 import React, { useCallback } from '@wordpress/element';
 
-const INSERTING_HOOK_NAME = 'isInsertingPageTemplate';
-const INSERTING_HOOK_NAMESPACE = 'automattic/full-site-editing/inserting-template';
+const INSERTING_HOOK_NAME = 'isInsertingPagePattern';
```

Comment:
👋🏼 Wanted to let you know that this rename broke the `from_template_selector` property in our tracking events. PR, for more context: https://github.com/Automattic/wp-calypso/pull/53640

I humbly recommend a more thorough code search in the future to catch any consumers of the existing hook 🙂

Reply:
Sorry about that! I did search for usage but clearly I wasn't thorough enough (I don't know how I missed it given the usage in `wpcom-block-editor` is in the same repo 🤦‍♂️). Losing 3 months worth of that prop is frustrating.